convert2rhel/unit_tests/subscription_test.py

```diff
@@ -333,3 +352,45 @@ def test_check_needed_repos_availability(self):
     def test_check_needed_repos_availability_no_repo_available(self):
         subscription.check_needed_repos_availability(["rhel"])
         self.assertTrue("rhel repository is not available" in logger.CustomLogger.warning.msg)
+
+    class VersionMocked(unit_tests.MockFunction):
+        def __init__(self, version):
+            self.version = version
+
+        def __call__(self, version):
+            return self.version
+
+    @unit_tests.mock(subscription.logging, "getLogger", GetLoggerMocked())
+    @unit_tests.mock(utils, "parse_version_number", VersionMocked(version={'major': '6', 'minor': '6'}))
+    def test_download_subscription_manager_packages_rhel66_or_less(self):
+        self.assertRaises(SystemExit, subscription.download_subscription_manager_packages)
+        self.assertTrue(len(subscription.logging.getLogger.error_msgs), 1)
+        self.assertTrue("Unable to download subscription-manager" in subscription.logging.getLogger.error_msgs[0])
+
+    class StoreContentMocked(unit_tests.MockFunction):
+        def __init__(self):
+            self.filename = None
+            self.content = None
+
+        def __call__(self, filename, content):
+            self.filename = filename
+            self.content = content
+            return True
+
+    @unit_tests.mock(utils, "store_content_to_file", StoreContentMocked())
+    @unit_tests.mock(utils, "parse_version_number", VersionMocked(version={'major': '6', 'minor': '7'}))
+    @unit_tests.mock(pkghandler, "call_yum_cmd", CallYumCmdMocked())
+    def test_download_subscription_manager_packages_rhel67(self):
```

Comment:
Add another test where major version is something we'd never encounter, such as `1`, and make sure we get logging